python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py

```diff
@@ -136,6 +137,8 @@ def transform_vp_timestamps(
             "vehicle_id",
         ],
     )
+    # TODO: review trip_id processing # pylint: disable=fixme
+    #  add more intelligent trip_id processing, this approach will randomly select trip_id record to keep
```

Comment:
The logic we are using to add `trip_id` is slightly random, it is functional but should be reviewed in the future to see exactly what `trip_id` we are pulling compared to the `trip_hash` grouping we are doing separately.